strings-csharp/StringFormattableString/StringFormattableStringTests/FormattableStringsUnitTests.cs

```diff
@@ -9,9 +11,10 @@ public class FormattableStringsUnitTests
         public void GivenAStringObject_WhenStringOperationsApplied_VerifyAccurateResults()
         {
             var stringObj = "Jane";
```

Comment:
This can go away as it doesn't really serve any purpose. A better way to write this test would be something like this:
`const string expectedStudentName = "John";`
`const int expectedStudentAge = 30;`
`var sampleString = _formattableStringMethods.StringExample(expectedStudentName, expectedStudentAge);`
`Assert.IsTrue(sampleString.Contains(expectedStudentAge);`
 etc.
Since we know exactly what string should be returned, since the method itself is building the string, we could even check just the whole expected string.